systemtest/src/test/java/io/strimzi/systemtest/operators/topic/TopicST.java

```diff
@@ -580,6 +587,77 @@ void testKafkaTopicChangingMinInSyncReplicas(ExtensionContext extensionContext)
         assertKafkaTopicStatus(topicName, namespace, NotReady, reason, reasonMessage, 2);
     }
 
+    /**
+     * @description This test case checks that Kafka cluster will not act upon KafkaTopic Custom Resources
+     * which are not of its concern, i.e., KafkaTopic Custom Resources are not labeled accordingly.
+     *
+     * @steps
+     *  1. - Deploy Kafka with short reconciliation time configured on Topic Operator
+     *     - Kafka is deployed
+     *  2. - Create KafkaTopic Custom Resource without any labels provided
+     *     - KafkaTopic Custom resource is created
+     *  3. - Verify that Kafka topic specified by created KafkaTopic is not created
+     *     - Given kafka topic is not present inside Kafka cluster
+     *  4. - Delete given KafkaTopic Custom Resource
+     *     - KafkaTopic Custom Resource is deleted
+     *
+     * @testcase
+     *  - topic-operator
+     *  - kafka-topic
+     *  - labels
+     */
+    @ParallelNamespaceTest
+    @KRaftNotSupported("TopicOperator is not supported by KRaft mode and is used in this test class")
+    void testTopicWithoutLabels(ExtensionContext extensionContext) {
+        final String namespaceName = StUtils.getNamespaceBasedOnRbac(namespace, extensionContext);
+        final String clusterName = mapWithClusterNames.get(extensionContext.getDisplayName());
+        final String scraperName = mapWithScraperNames.get(extensionContext.getDisplayName());
+        final String kafkaTopicName = "topic-without-labels";
```

Comment:
Here you should you `TestStorage` class and then retrieve all these fields from that instance. (I know that you just copy-pasted such code but It just small change, which does not affect anything)